### PR TITLE
Map changes for 1.13

### PR DESCRIPTION
**Pull request by PundaChen**
Some more changes, I'll try to get the rest of the maps jsons finished today as well

### CODE REVIEW

File: CTW/DesertCountry/map.json [hunk 1]
```diff
@@ -29,24 +29,28 @@
...
+				"color": "dark green",
+				"woolcolor": "green",
 				"teams": ["blue"],
 				"region": {"min": "535, 11, -329", "max": "535, 11, -329"}
 			},
 			{
 				"name": "Purple",
-				"color": "purple",
+				"color": "dark purple",
+				"woolcolor": "purple"
```
Comment (BennyDoesTheStuff) [inline CTW/DesertCountry/map.json]: You’re missing a comma here

File: CTW/Race_for_Victory_2/map.json [hunk 2]
```diff
@@ -28,26 +28,30 @@
 		"wools": [
 			{
 				"name": "Purple",
-				"color": "purple",
+				"color": "dark purple",
+				"woolcolor": "magenta"
```
Comment (BennyDoesTheStuff) [inline CTW/Race_for_Victory_2/map.json]: Comma missing here

File: CTW/Race_for_Victory_2/map.json [hunk 3]
```diff
@@ -28,26 +28,30 @@
...
+				"woolcolor": "yellow",
 				"teams": ["blue"],
 				"region": {"min": "1, 5, -64", "max": "1, 5, -64"}
 			},
 
 			{
 				"name": "Lime",
-				"color": "lime",
+				"color": "green",
+				"woolcolor": "lime"
```
Comment (BennyDoesTheStuff) [inline CTW/Race_for_Victory_2/map.json]: Comma missing here

File: CTW/TechnoWar/map.json [hunk 4]
```diff
@@ -28,25 +28,29 @@
...
-				"color": "orange",
+				"color": "gold",
+				"woolcolor": "orange",
 				"teams": ["blue"],
 				"region": {"min": "-1, 57, 92", "max": "-1, 57, 92"}
 			},
 			{
 				"name": "Yellow",
 				"color": "yellow",
+				"woolcolor": ""yellow",
```
Comment (BennyDoesTheStuff) [inline CTW/TechnoWar/map.json]: You added an extra quote here

**Comment by PundaChen** [comment]
Should be corrected now, thanks

**Comment by TRose2000** [comment]
1.14 when :eyes:

**Comment by BennyDoesTheStuff** [comment]
1.14 isn’t a big technical update so there’s barely anything that would need to be changed. Additionally, a stable 1.14 Spigot isn’t even out yet

**Comment by TRose2000** [comment]
Spigot 1.14 is out already?

**Comment by PundaChen** [comment]
Added more maps to the pr, almost done ^^

**Comment by TRose2000** [comment]
> Added more maps to the pr, almost done ^^

ur doing a great job!

**Pull request merged**